packages/authgear-react-native/src/index.ts

```diff
@@ -945,6 +1014,16 @@ export class ReactNativeContainer {
     await this.storage.setBiometricKeyID(this.name, kid);
   }
 
+  /**
+   * Authenticate with biometric authentication.
+   * Platform specific biometric authentication UI will be shown.
+   * You may first enable biometric authentication in Authgear Portal (Your Project \> Authentication \> Biometric \> Enable biometric authentication).
+   * Otherwise, `BiometricDisallowed` error will be thrown.
```

Comment:
We do not have `BiometricDisallowed` error. Remove this line.

Reply:
ok